src/api/app/controllers/webui/staging_workflows_controller.rb

```diff
@@ -4,7 +4,7 @@ class Webui::StagingWorkflowsController < Webui::WebuiController
   before_action :require_login, except: [:show]
   before_action :set_bootstrap_views
   before_action :set_project, only: [:new, :create]
-  after_action :verify_authorized, except: [:show, :new, :destroy] #TODO: authorization for destroy
```

Comment:
@krauselukas That's regarding the commit message of this PR (though I need to refer this via the commit diff)... Please always add a newline between commit message subject and body. Otherwise it looks strange in the git client.